gmagickpixel_methods.c

```diff
@@ -129,7 +149,7 @@ PHP_METHOD(gmagickpixel, getcolor)
 PHP_METHOD(gmagickpixel, getcolorcount)
 {
 	php_gmagickpixel_object *internp;
-	long color_count;
+	zend_long color_count;
 	
```

Comment:
just fyi, that one isn't necessary to be zend_long, it won't hurt, but isn't required.

The issue with `zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "l", &color_count)` is that zpp will be writing 64bits of data into the memory location pointed to by `&color_count`.

On Windows 64bit long is 32bit, and so doing that the write by zpp would touch memory outside of the variable.

However for getColorCount() there is no "writing a variable into memory", instead getColorCount is just returning a value of type `unsigned long`, and there is no issue with the code casting a 32bit long to a 64bit long.

The unsigned-ness could be an issue - the compiler should be issuing a warning there.....oh, apparently GCC doesn't do that by default and we need to enable -Wconversion....